### PR TITLE
 Remove post it

### PR DESCRIPTION
**Pull request by stef74**
Remove post it for area and add in information block

Before :  
capture d ecran 2017-12-21 a 21 38 29

After : 
capture d ecran 2017-12-21 a 21 37 59

### CODE REVIEW

**Comment by brunobesson** [comment]
Relates to #1884

**Comment by brunobesson** [comment]
Looks like your PR breaks the tests. Did you have a look at it, or do you need help on the subject?

File: c2corg_ui/templates/outing/view.html [hunk 1]
```diff
@@ -106,12 +106,6 @@ <h1 class="routes">
       </h1>
     </header>
 
-  % if outing.get('areas'):
-    <article class="location-static">
-      ${show_areas(outing)}
```
Comment (brunobesson) [inline c2corg_ui/templates/outing/view.html]: Shouldn't the `show_areas` partial be removed now? Or are postits used somewhere else?

Comment (stef74) [reply c2corg_ui/templates/outing/view.html]: Please look title ... WIP ;-) ... i'm currently working on it !

Comment (stef74) [reply c2corg_ui/templates/outing/view.html]: but no show area it's allready use ;-)

**Comment by stef74** [comment]
@brunobesson now you can review :-p

File: c2corg_ui/templates/outing/view.html (hunk 1 above)
Comment (brunobesson) [reply c2corg_ui/templates/outing/view.html]: Mmh. Worry, didn't see it is a WIP. But why did you request my review then? (or did I invent it?)

Comment (stef74) [reply c2corg_ui/templates/outing/view.html]: Now is finish ;-)

**Comment by stef74** [comment]
On other doc the post-it not hide some information (except for extrem size). But I can do in other PR

**Comment by brunobesson** [comment]
I think it does, I have tested some waypoints (withou image?) and my sceeen is really standard. I would appreciate if we could get rid of all post-its.

**Pull request merged**